### PR TITLE
Update how-to-connect-to-an-sms-provider-in-configuration-manager-by-…

### LINKED ISSUE #1474: Broken Links (how-to-connect-to-an-sms-provider-in-configuration-manager-by-using-wmi.md)
In File:
SCCMdocs\sccm\develop\core\understand\how-to-connect-to-an-sms-provider-in-configuration-manager-by-using-wmi.md

The following links are dead and I could not find a suitable replacement:

http://go.microsoft.com/fwlink/?LinkId=44022
http://go.microsoft.com/fwlink/?LinkId=44023

### PR DESCRIPTION
**Pull request by AndrewJNet**
Updated broken/msdn/go.microsoft.com links with docs.microsoft.com links. Also fixes #1474.

#MMS2019Docathon

### Summarize the change in the pull request title

Updated broken/msdn/go.microsoft.com links with docs.microsoft.com links. Also fixes #1474. 

Fixes #1474

### CODE REVIEW

**Comment by aczechowski** [comment]
Thanks @AndrewJNet for the contribution. You'll need to sign the CLA. Otherwise, I'll review this after the end of the MMS contest.

#hold-off
(Reviewers, please don't edit, I'll take care of it.)

**Comment by PRMerger17** [comment]
@AndrewJNet : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

**Comment by AndrewJNet** [comment]
@aczechowski All set!

**Comment by rjagiewich** [comment]
@aczechowski  - Would you [add a hash comment](https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-pull-request-automation?branch=master) like `#sign-off` to indicate the action to take on this PR?  Thanks.

File: sccm/develop/core/understand/how-to-connect-to-an-sms-provider-in-configuration-manager-by-using-wmi.md [hunk 1]
```diff
@@ -14,28 +14,28 @@ ms.collection: M365-identity-device-management
...
 Before connecting to the SMS Provider for a local or remote System Center Configuration Manager site server, you first need to locate the SMS Provider for the site server. The SMS Provider can be either local or remote to the Configuration Manager site server you are using. The Windows Management Instrumentation (WMI) class `SMS_ProviderLocation` is present on all Configuration Manager site servers, and one instance will contain the location for the Configuration Manager site server you are using.  
 
- You can connect to the SMS Provider on a Configuration Manager site server by using the WMI [SWbemLocator](http://go.microsoft.com/fwlink/?LinkId=44022) object or by using the Windows Script Host `GetObject` method. Both approaches work equally well on local or remote connections, with the following limitations:  
+ You can connect to the SMS Provider on a Configuration Manager site server by using the WMI [SWbemLocator](https://docs.microsoft.com/en-us/windows/desktop/wmisdk/swbemlocator) object or by using the Windows Script Host `GetObject` method. Both approaches work equally well on local or remote connections, with the following limitations:
```
Comment (aczechowski) [inline sccm/develop/core/understand/how-to-connect-to-an-sms-provider-in-configuration-manager-by-using-wmi.md]: Remove locale (`en-us/`) from all URLs

**Comment by aczechowski** [comment]
Ping @AndrewJNet can you review and revise?

**Comment by PRMerger20** [comment]
@AndrewJNet : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

**Comment by PRMerger20** [comment]
@AndrewJNet : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

**Comment by aczechowski** [comment]
Thanks @AndrewJNet, looks good 😄 
#sign-off 
*Once merged*, this should be live with the next daily publishing process. Right now that *should* be tomorrow (Wednesday, 22 May) around 11:00 Pacific time. No further action needed from you.

**Pull request merged**